packages/node/src/methods.ts

```diff
@@ -3,18 +3,23 @@ import { PRODUCTS } from '@xkeys-lib/core'
 import * as HID from 'node-hid'
 import { NodeHIDDevice } from './node-hid-wrapper'
 
-import { isHID_Device, isHID_HID } from './lib'
+import { isHID_Device } from './lib'
 
 import { HID_Device } from './api'
 
-/** Sets up a connection to a HID device (the X-keys panel) */
+/**
+ * Sets up a connection to a HID device (the X-keys panel)
+ *
+ * If called without arguments, it will select any connected X-keys panel.
+ */
 export function setupXkeysPanel(): Promise<XKeys>
 export function setupXkeysPanel(HIDDevice: HID.Device): Promise<XKeys>
-export function setupXkeysPanel(HIDDevice: HID.HID): Promise<XKeys>
 export function setupXkeysPanel(devicePath: string): Promise<XKeys>
-export async function setupXkeysPanel(devicePathOrHIDDevice?: HID.Device | HID.HID | string): Promise<XKeys> {
+export async function setupXkeysPanel(
```

Comment:
@Julusian  FYI, I did some testing where passing a `HID.HID` or `HID.HIDAsync` into `setupXkeysPanel`, but I ran into problems because neither exposed any `devicePath` (contrary to what I'd previously written in a code comment in `isHID_HID`).

I decided to remove the support for passing `HID.HID` or `HID.HIDAsync` into `setupXkeysPanel` altogether instead.

Reply:
I don't have any idea where that `devicePath` property would have come from, its not something that node-hid ever supported. Although there is a method to retrieve the path, vid and pid for an opened hid device now, so it could be rewritten if wanted?